### PR TITLE
The menu toggle is now visible on all the time

### PR DESCRIPTION
**Pull request by eloraju**

### CODE REVIEW

File: src/components/toggle.module.css [hunk 1]
```diff
@@ -48,6 +49,7 @@
 
 .active > span:nth-child(2) {
   background: transparent;
+  backdrop-filter: unset;
```
Comment (midori0507) [inline src/components/toggle.module.css:52]: Can you share reason for using this?
Cause i check this from caniuse, Firefox doenst support this tag

Comment (midori0507) [reply src/components/toggle.module.css:52]: https://caniuse.com/?search=backdrop-filter

Comment (eloraju) [reply src/components/toggle.module.css:52]: That's me being lazy 😅
Addu linked me a site that used `backdrop-filter` to create the inversion effect and I just rolled with that. The reason for the inversion is that we want to add content with black background color to the site. Making the menu be inverse of the background makes it stand out all the time.

I'll check something else out. `mix-blend-mode` should work more or less the same way :thinking:

Comment (midori0507) [reply src/components/toggle.module.css:52]: I would say just set normal background css is easier and no cross-browser issue then? :D 
Btw you can free to merge if you think PR is good enough, doesnt need to have my or anyone approval tho

**Pull request merged**